swift/Tooling/Templates/Workflow (Verbose).xctemplate/___FILEBASENAME___Workflow.swift

```diff
@@ -42,7 +42,7 @@ extension ___VARIABLE_productName___Workflow {
 
         typealias WorkflowType = ___VARIABLE_productName___Workflow
 
-        func apply(toState state: inout ___VARIABLE_productName___Workflow.State) -> ___VARIABLE_productName___Workflow.Output? {
+        func apply(toState state: inout ___VARIABLE_productName___Workflow.State, context: inout SideEffectContext<___VARIABLE_productName___Workflow>) -> ___VARIABLE_productName___Workflow.Output? {
```

Comment:
In addition to updating the methods affected by this, we should likely add the definition of `SideEffect` and `perform` to the template (since it's the "verbose" one).